demos/k8s-demo/01_security_admin_steps

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+. ./utils.sh
+. ./admin_config.sh
+
+##################################################
+step "Create a new namespace"
+
+delete_ns_and_cleanup "quick-start-backend-ns"
+kubectl create namespace "quick-start-backend-ns"
+
+##################################################
+step "Add certificates to Kubernetes Secrets"
+
+# add pg certificates to kubernetes secrets
+kubectl --namespace quick-start-backend-ns \
+  create secret generic \
+  quick-start-backend-certs \
+  --from-file "etc/pg_server.crt" \
+  --from-file "etc/pg_server.key"
+
+##################################################
+step "Create StatefulSet for Database"
+
+kubectl --namespace quick-start-backend-ns apply \
+  --filename "etc/pg.yml"
+
+wait_for_app "quick-start-backend" "quick-start-backend-ns"
+# why doesn't this work?
```

Comment:
do we want to leave this comment? maybe we should just log a gh issue with "update tutorial flows to use `rollout status`"

Reply:
https://github.com/cyberark/secretless-broker/issues/657